docker/docker-compose.yml.example

```diff
@@ -0,0 +1,46 @@
+version: '2.4'
+
+
+services:
+
+  web:
+    build:
+      context: .
+    ports:
+      - 80:80
+    working_dir: /app
+    volumes:
+      - ..:/app
+    env_file:
+      - ../.env
+    environment:
+      - WEB_DOCUMENT_ROOT=/app/public
+      - WEB_DOCUMENT_INDEX=index.php
+      - XDEBUG_REMOTE_HOST=${XDEBUG_REMOTE_HOST:-localhost}
+
+  mysql:
+    image: mysql:5.7
+    volumes:
+      - mysql_data:/var/lib/mysql
+    env_file:
+      - ../.env
+    environment:
+      - MYSQL_ROOT_PASSWORD=$DB_PASSWORD
+      - MYSQL_DATABASE=$DB_DATABASE
+
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin:latest
+    ports:
+      - 8068:80
+    env_file:
+      - ../.env
+    environment:
+      - PMA_HOST=$DB_HOST
+      - PMA_USER=$DB_USERNAME
+      - PMA_PASSWORD=$DB_PASSWORD
+
+
```

Comment:
I would remove extra white line (at the beginning of the file too) just to clean it up